docs/index.md

```diff
@@ -268,6 +268,16 @@ precondition if: -> { subject.active? } do
 end
 ```
 
+Failing one precondition doesn't stop the execution. So if you have multiple conditions defined all of the would be checked. To stop propagation use `next` like this: 
+
+```
+precondition do
+  next decline_with(:failing_first) unless first_condition_is_false?
+  break decline_with(:failing_second) unless first_condition_is_true_but_second_is_false?
```

Comment:
What does the `break` do here?

Reply:
It is an example. NP - the docs are rejected by the Granite guys